cmd/commands.go

```diff
@@ -17,6 +17,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
```

Comment:
Can you please move this import lower, in the "non std-lib" import group?

Reply:
sure thanks corrected